### ISSUE TITLE
[Usable - Windows Powershell]: Alt keys should be defined for the tab items so that keyboard user can directly switch to any tab.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8481]
**User Experience:**
All the users using keyboard would be impacted as if there are 15 tabs open and currently 3rd tab is selected and user has to navigate to 10th tab, then user has to navigate each tab coming between than and has to press multiple times the key 'Ctrl+Tab' or 'Ctrl+Shift+Tab' to navigate.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Open multiple tab using 'Ctrl+Shift+T'.
3. Now navigate using 'Tab' key or press 'Alt' key to switch directly from one tab to other specific.
4. Observe the issue.

**Actual Result:**
Alt keys are not defined for the tab items, so if there are 15 tabs open and currently 3rd tab is selected and user has to navigate 10th tab, then user has to navigate each tab coming between than and has to press multiple times the key 'Ctrl+Tab' or 'Ctrl+Shift+Tab' to navigate.

**Expected Result:**
Alt keys should be defined for the tab items so that keyboard user can directly switch to any tab.

[Usable_Tabs should be accessible via keyboard.zip](https://github.com/microsoft/terminal/files/5636232/Usable_Tabs.should.be.accessible.via.keyboard.zip)

**Comment by zadjii-msft** [comment]
Tabs 1-9 are bound to <kbd>ctrl+alt+1</kbd>-<kbd>ctrl+alt+9</kbd>, Do we need more than that? I'm fairly certain most tabbed applications stop at 9

**Comment by DHowett** [comment]
Correct.

The only QOL improvement here is that in many browsers, tab "9" (or tab action 9) is "jump to _last_ tab"

@Chandresh1007 it is not normal for tabbed applications to offer a keyboard shortcut for more than 9 tabs. Please clarify your intent.

**Comment by Chandresh1007** [comment]
@DHowett Yes, if keys are defined to switch between tabs then we are good to close this bug.
